http_client.rst

```diff
@@ -631,6 +631,15 @@ according to the ``multipart/form-data`` content-type. The
         'body' => $formData->bodyToIterable(),
     ]);
 
+HttpClient will stream the upload of the body by default. This might not work with all
```

Comment:
What about embedding this in the code block above like this?

```
// if the server is not compatible with streaming the request
// body (e.g. result in a 411 response status code), send
// the body as a string:
$client->request('POST', 'https://...', [
    'headers' => $formData->getPreparedHeaders()->toArray(),
    'body' => $formData->bodyToIterable(),
]);
```

Reply:
* I added more details of the error message on purpose, for search engines - cause that's how people will notice the problem in the first place.
* `bodyToString()` needs to be mentioned somewhere explicitly! Right now, it's not documented at all.

Sure I could integrate it into the code block, but I always find it odd if *one* line of code is preceded by *five* lines of comments... ;-)